### ISSUE TITLE
Publishing metadata to Språkbanken Text resource catalog

### ISSUE THREAD

**Issue by arildm** [spraakbanken/mink-backend#98]
As an entrusted user, I want to make Mink publish my uploaded metadata/data to the Språkbanken Text resource catalog at https://spraakbanken.gu.se/en/resources

I think this involves a new job type. It would be applicable to Metadata resources, not Corpus resources. I guess it should commit the metadata yaml file to the [spraakbanken/metadata](https://github.com/spraakbanken/metadata/) repo. For that to be handled properly, some change may be needed in [spraakbanken/metadata-api](https://github.com/spraakbanken/metadata-api/) as well.

Related to:
- #82 
- #84

**Comment by arildm** [comment]
I discussed this with @MartinHammarstedt a few weeks back. I think we were reconsidering the workflow for publishing metadata, but I cannot really remember what we came up with instead. I just remember it did _not_ involve the Metadata resource type... This needs further discussion!

**Comment by anne17** [comment]
(This is one of the remaining tasks of https://github.com/spraakbanken/mink-backend/issues/82)

Let me rephrase the issue a little so it fits the current state of the project:

We need functionality for publishing metadata to our [resource catalogue](https://spraakbanken.gu.se/resurser). This do one of the following:

1. rsync to a dir on one of our servers (to a dir named after the user's org prefix) from where it will be checked into our [metadata git repo](https://github.com/spraakbanken/metadata)
2. checked into our metadata git repo directly
3. checked into the organisation's own git repo.

Option 3 is probably the preferred way, especially since there are plans for extending the metadata API to accept multiple repositories which should be administered by the different Språkbanken organisations.

Only entrusted users with an accepted org prefix should be able to do this.
